### ISSUE TITLE
Is it possible to create a github release?

### ISSUE THREAD

**Issue by damianesteban** [changesets/action#166]
Is it possible to create a github release with this action or with changesets in general? Thank you.

**Comment by Andarist** [comment]
Yes, this action automatically creates releases (if you don't disable that). We use this in this very repository and this is our last release: https://github.com/changesets/action/releases/tag/v1.2.2

**Comment by markdalgleish** [comment]
Does it need a `GITHUB_TOKEN` env var to work? Seems to silently fail if this is missing.

**Comment by Jack-Works** [comment]
OK, I figured it out.

https://github.com/changesets/action/blob/main/src/run.ts#L100

It scans the publish command, you need to include "New tag:" in stdout, which is not intuitive...
